src/__tests__/lib/exampleData.js

```diff
@@ -621,47 +621,92 @@ export const action = Object.freeze({
       muted_topics: [],
       muted_users: [],
       presences: {},
+      development_environment: false,
+      event_queue_longpoll_timeout_seconds: 600,
+      max_avatar_file_size_mib: 3,
+      max_file_upload_size_mib: 3,
       max_icon_file_size: 3,
+      max_icon_file_size_mib: 3,
+      max_logo_file_size_mib: 3,
+      password_min_length: 8,
+      password_min_guesses: 3,
+      realm_add_custom_emoji_policy: 3,
       realm_add_emoji_by_admins_only: true,
       realm_allow_community_topic_editing: true,
       realm_allow_edit_history: true,
       realm_allow_message_deleting: true,
       realm_allow_message_editing: true,
       realm_authentication_methods: { GitHub: true, Email: true, Google: true },
       realm_available_video_chat_providers: {},
+      realm_avatar_changes_disabled: false,
       realm_bot_creation_policy: 3,
       realm_bot_domain: 'example.com',
-      realm_create_stream_by_admins_only: true,
+      realm_community_topic_editing_limit_seconds: 600,
+      realm_create_private_stream_policy: 3,
+      realm_create_public_stream_policy: 3,
+      realm_create_web_public_stream_policy: 3,
```

Comment:
This example action should probably get a comment at the top warning these values may not be representative, like we have on users and messages.

(This particular property is a fun one because apparently the allowed values are 2, 4, 6, and 7. But it's fine to have 3 here! If we ever write a test that cares, we'll notice then.)